src/components/LinearProgressBar/LinearProgressBar.st.css

```diff
@@ -0,0 +1,80 @@
+:import {
+  -st-from: "wix-ui-core/dist/src/components/LinearProgressBar/LinearProgressBar.st.css";
+  -st-default: LinearProgressBar;
+}
+
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: progress, error, main, danger, mainMutedHover, dangerMutedHover, backgroundPrimary;
+}
+
+:import {
+  -st-from: "../../colors.st.css";
```

Comment:
The components should not really use `colors.st.css`, only `palette.st.css`. If there is no alias for WHITE in palette file, add one please.